src/ansiblelint/_prerun.py

```diff
@@ -29,18 +29,39 @@
 """
 
 
-def check_ansible_presence() -> None:
-    """Assures we stop execution if Ansible is missing."""
+def ensure_ansible_presence() -> str:
+    """Assures we stop execution if Ansible is missing or outdated."""
     failed = False
+    ansible_cli_version = ""
+    result = subprocess.run(
+        args=["ansible", "--version"],
+        stdout=subprocess.PIPE,
+        universal_newlines=True,
+        check=False,
+    )
+    if result.returncode != 0:
+        print(
+            "FATAL: Unable to retrieve ansible cli version: %s" % result.stdout,
+            file=sys.stderr,
+        )
+        sys.exit(ANSIBLE_MISSING_RC)
+    match = re.match(r"^ansible ([^\s]+)", result.stdout)
+    if not match:
+        print(
+            "FATAL: Unable parse ansible cli version: %s" % (result.stdout),
+            file=sys.stderr,
+        )
+        sys.exit(ANSIBLE_MISSING_RC)
+    ansible_cli_version = match.group(1)
     try:
         # pylint: disable=import-outside-toplevel
-        from ansible import release
+        from ansible.release import __version__ as ansible_module_version
 
-        if version.parse(release.__version__) <= version.parse(ANSIBLE_MIN_VERSION):
+        if version.parse(ansible_module_version) < version.parse(ANSIBLE_MIN_VERSION):
             failed = True
     except (ImportError, ModuleNotFoundError, UnboundLocalError) as e:
```

Comment:
UnboundLocalErrror probably should not be trapped here, as it should only show up from a coding error, and having it caught in an exception handler could make it harder to track down the issue.